reliability/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pyyaml
```

Comment:
I think it would be a good idea to lock the version for each of these - as unwanted updates can break things. Something we always followed in our previous team.

Reply:
@kedark3 Good point! Thanks for sharing your experience.

Reply:
@kedark3 PTAL https://github.com/openshift/svt/pull/677/commits/65b1fd678d96caf724ef577c41c238020561c716

Reply:
that looks good @qiliRedHat